packages/core-typings/src/Apps.ts

```diff
@@ -10,21 +10,42 @@ export type AppScreenshot = {
 	modifiedAt: string;
 };
 
-export type PricingPlan = {
+export type AppTiers = {
+	perUnit: boolean;
+	minimum: number;
+	maximum: number;
+	price: number;
+};
+
+export type AppPricingPlan = {
 	id: string;
 	enabled: boolean;
 	price: number;
 	trialDays: number;
 	strategy: string;
 	isPerSeat: boolean;
-	tiers?: Tiers[];
+	tiers?: AppTiers[];
 };
 
-export type Tiers = {
-	perUnit: boolean;
-	minimum: number;
-	maximum: number;
-	price: number;
+export type AppLicense = {
+	license: string;
+	version: number;
+	expireDate: string;
+};
+
+export type AppSubscriptionInfo = {
+	typeOf: string;
+	status: string;
+	statusFromBilling: boolean;
+	isSeatBased: boolean;
+	seats: number;
+	maxSeats: number;
+	license: AppLicense;
+	startDate: string;
+	periodEnd: string;
+	endDate: string;
+	externallyManaged: boolean;
```

Comment:
```suggestion
```
This property will not be exposed in the near future.

Reply:
Solving it on the next commit.